modelcheck/src/main/kotlin/de/itemis/mps/gradle/modelcheck/ModelCheck.kt

```diff
@@ -222,34 +222,28 @@ private fun oneTestCasePerMessage(item: IssueKindReportItem, model: SModel, proj
 private fun oneTestCasePerModel(models: Iterable<SModel>, errorsPerModel: Map<SModel, List<IssueKindReportItem>>, project: Project): List<Testcase> {
     return models.map {
         val errors = errorsPerModel.getOrDefault(it, emptyList())
-        fun reportItemToContent(s: Failure, item: IssueKindReportItem): Failure {
-            return when (val path = item.path) {
-                is IssueKindReportItem.PathObject.ModelPathObject -> {
-                    val model = path.resolve(project.repository)!!
-                    val message = "${item.message} [${model.name.longName}]"
-                    Failure(
-                        message = "${s.message}\n $message",
-                        type = s.type
-                    )
+        val message = buildString {
+            for (item in errors) {
+                when (val path = item.path) {
+                    is IssueKindReportItem.PathObject.ModelPathObject -> {
+                        val model = path.resolve(project.repository)!!
+                        append("\n ${item.message} [${model.name.longName}]")
```

Comment:
Let's skip the separator (`\n `) on the first item.

Reply:
👍 
I was duplicating the previous output, but changed it now.